app/controllers/cloud_tenant_controller.rb

```diff
@@ -77,9 +77,10 @@ def create
     assert_privileges("cloud_tenant_new")
     case params[:button]
     when "cancel"
-      javascript_redirect :action => 'show_list',
```

Comment:
When cleaning up this file, can you, please, also cleanup the i18n a bit?

I mean: in cases where there's just one value for the `%{model}`, it makes no sense to do the UI lookup and the expansion. 

Can you replace the `format` call  by a constant?

It will make @mzazrivec and the translators happy.

The same applies on several places in this file.

Thx!